bin/projects/dbatools/dbatools/Utility/DbaDate.cs

```diff
@@ -0,0 +1,237 @@
+﻿using System;
+
+namespace Sqlcollaborative.Dbatools
+{
+    namespace Utility
+    {
+        /// <summary>
+        /// A dbatools-internal datetime wrapper for neater display
+        /// </summary>
+        public class DbaDate : DbaDateTimeBase
+        {
+            #region Constructors
+            /// <summary>
+            /// Constructs a generic timestamp object wrapper from an input timestamp object.
+            /// </summary>
+            /// <param name="Timestamp">The timestamp to wrap</param>
+            public DbaDate(DateTime Timestamp)
+            {
+                _timestamp = Timestamp;
+            }
+
+            /// <summary>
```

Comment:
I think you should kill the empty comment blocks. Add them later when you have actually comments. Is there a reason you are creating empty ones?